images/retag-and-push/action.yml

```diff
@@ -0,0 +1,36 @@
+name: Retag and push images
+description: |
+  This action will, as stated in its name, retag and push images
+
+inputs:
+  src-image:
+    description: |
+      The image that will be retagged
+    required: true
+  dst-image:
+    description: |
+      A target image that will be generated by retagging src-image and later
+      pushed
+    required: true
+  username:
+    description: |
+      Username used to login to the repository the push is aimed at
+    required: true
+  password:
+    description: |
+      Password used to login to the repository the push is aimed at
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - uses: docker/login-action@v2
+      with:
+        registry: quay.io
+        username: ${{ inputs.username }}
+        password: ${{ inputs.password }}
+
+    - shell: bash
+      run: |
+        docker tag ${{ inputs.src-image }} ${{ inputs.dst-image }}
```

Comment:
Let's keep it similar to the other actions: https://github.com/stackrox/actions/blob/main/release/wait-for-image/action.yml#L21-L30 (even if it _just_ two commands, using a script simplifies testing of the action.